static/styles/compose.css

```diff
@@ -47,7 +66,7 @@
     border-top: 1px solid hsla(0, 0%, 0%, 0.07);
     transition: background-color 200ms linear;
 
-    padding: 8px 10px 8px 10px;
+    padding: 4px;
```

Comment:
What's the story with this change?  It looks like it would affect the open compose state.

Reply:
Note this does not affect the open compose state. Greater bottom padding also added in later revision.